test/servico/cst01Test.java

```diff
@@ -55,7 +55,7 @@ public void tearDown() {
     public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Funcionario1", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 82).](https://www.codacy.com/app/calebepb/substituicao/file/2738501511/issues/source?bid=3308597&fileBranchId=3353492#l58)